org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationMatching.cs

```diff
@@ -0,0 +1,718 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using Quartz;
+using Rock;
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Web.Cache;
+using Newtonsoft.Json;
+using System.Net;
+using OfficeOpenXml;
+using System.Drawing;
+using OfficeOpenXml.Style;
+using RestSharp;
+using Rock.Security;
+using System.Diagnostics;
+using System.ComponentModel;
+using System.Threading;
+using OfficeOpenXml.FormulaParsing.LexicalAnalysis;
+using System.Text;
+using DotLiquid.Tags;
+
+namespace org.crossingchurch.HubspotIntegration.Jobs
+{
+    /// <summary>
+    /// Job to supply hubspot contacts with a rock_id to the pull related information.
+    /// </summary>
+    [DisplayName("Hubspot Integration: Match Records")]
+    [Description("This job only supplies Hubspot contacts with a Rock ID and adds potential matches to an excel spreadsheet for further investigation.")]
+    [DisallowConcurrentExecution]
+
+    [TextField("AttributeKey", "The attribute key for the global attribute that contains the HubSpot API Key. The attribute must be encrypted.", true, "HubspotAPIKeyGlobal")]
+    [TextField("Business Unit", "Hubspot Business Unit value", true, "0")]
+    [TextField("Potential Matches File Name", "Name of the file for this job to list potential matches for cleaning", true, "Potential_Matches")]
+    public class HubspotIntegrationMatching : IJob
+    {
+        private string key { get; set; }
+        private List<HSContactResult> contacts { get; set; }
+        private int request_count { get; set; }
+        private string businessUnit { get; set; }
```

Comment:
The names of these properties violate naming rules. Please update them.
- `key` -> `Key`
- `contacts` -> `Contacts`
- `request_count` -> `RequestCount`

And `businessUnit` has 0 references. So can we remove it?